src/extension.js

```diff
@@ -41,54 +41,64 @@ const getConfig = () => {
   };
 };
 
+const createPanel = async context => {
+  const html = await readHtml(path.resolve(context.extensionPath, 'webview/index.html'));
+
+  const panel = vscode.window.createWebviewPanel('codesnap', 'CodeSnap 📸', vscode.ViewColumn.Two, {
+    enableScripts: true,
+    localResourceRoots: [vscode.Uri.file(context.extensionPath)]
+  });
+  panel.webview.html = html;
+
+  return panel;
+};
+
+let lastUsedImageUri = vscode.Uri.file(path.resolve(homedir(), 'Desktop/code.png'));
+const saveImage = async data => {
+  const uri = await vscode.window.showSaveDialog({
+    filters: { Images: ['png'] },
+    defaultUri: lastUsedImageUri
+  });
+  lastUsedImageUri = uri;
+  uri && writeFile(uri.fsPath, Buffer.from(data, 'base64'));
+};
+
+const copyImage = async data => {
+  const [err, stdout, stderr] = await copyImg(Buffer.from(data, 'base64'));
+  if (err) {
```

Comment:
this one is just a minor idea but could do early return here might be a bit cleaner. I personally like early return style when the logic gets isolated into a function.